networkconfig/local-testnet.go

```diff
@@ -0,0 +1,17 @@
+package networkconfig
+
+import (
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv/protocol/v2/blockchain/beacon"
+)
+
+var LocalTestnet = NetworkConfig{
+	Name:                 "local-testnet",
+	Beacon:               beacon.NewLocalTestNetwork(spectypes.PraterNetwork),
+	Domain:               spectypes.DomainType{0x0, 0x0, 0x4, 0x1},
```

Comment:
Is it intended that the domain is the same as in `jato-v2`?

Reply:
@guym-blox shouldn't be an issue since this is a closed network, but I'd give it its own domain anyway.